cmd/fees/config/config.go

```diff
@@ -11,8 +11,15 @@ import (
 	"github.com/vultisig/plugin/storage"
 )
 
+type verifier struct {
+	URL         string `mapstructure:"url"`
+	Token       string `mapstructure:"token"`
+	PartyPrefix string `mapstructure:"party_prefix"`
+}
```

Comment:
I feel like this overlaps with another verifier config - non-blocking, largely up to you on how you want to handle this, but maybe worth checking later